src/enrollments/EnrollmentCollectionEntity.js

```diff
@@ -16,4 +16,12 @@ export class EnrollmentCollectionEntity extends Entity {
 		const enrollmentEntities = this._entity.getSubEntitiesByRel(Rels.userEnrollment);
 		return enrollmentEntities.map(e => e.href).filter(href => href);
 	}
+
+	getActionByName(url) {
+		if (!this._entity || !this._entity.hasActionByName(url)) {
```

Comment:
maybe use actionName instead of url?

Reply:
actionName could also make sense.

Reply:
yea, both works, i'm not sure which is better now